src/bicep-registry/scheduled-action/README.md

```diff
@@ -0,0 +1,19 @@
+# Cost Management scheduled action
+
+Creates a Cost Management scheduled action to notify recipients when an anomaly is detected or on a recurring schedule.
+
+## Description
+
+Use this module within other Bicep templates to simplify creating and updating scheduled actions.
+
+## Parameters
+
+| Name | Type | Required | Description |
+| :--- | :--: | :------: | :---------- |
+
+## Outputs
+
+| Name | Type | Description |
+| :--- | :--: | :---------- |
+
+## Examples
```

Comment:
Examples are added as part of the local build process. There's another PR tracking that script.